index.js

```diff
@@ -58,7 +58,11 @@ class ClinicDoctor {
     })
 
     proc.once('exit', function (code, signal) {
-      // the process did not exit normally
+      // Windows uncaught SIGINT has exit code 3221225786
```

Comment:
Could you document it like you described in the issue? The `exit` code isn't actually `SIGINT` but rather `STATUS_CONTROL_C_EXIT`. Also link to https://msdn.microsoft.com/en-us/library/cc704588.aspx and use the hexadecimal notation like they do in the MSDN docs (`0xC000013A`).

Reply:
Done in new commit